### PR TITLE
[AD-329] Add GUI usage guide

### PR DESCRIPTION
**Pull request by maksbotan**
**Description:** GUI usage guide

**YT issue:** https://issues.serokell.io/issue/AD-329

**Checklist:**

- [x] Updated docs if necessary
  - [x] [README](README.md)
  - [x] [TUI usage guide](docs/usage-tui.md)
  - [x] [Configuration documentation](docs/configuration.md)
- [x] My code complies with the [style guide](docs/code-style.md)
- [x] Tested my changes if they modify code

### CODE REVIEW

File: docs/usage-gui.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Ariadne GUI usage guide
+
+Ariadne GUI is made with Qt framework and is a completely native application that looks and feels
+the same on Linux, Windows and macOS. This document will present you the most of its features.
+
+## Overview
+
+When you launch `ariadne-qt` the main window appears.
+
+**TODO** screenshot
```
Comment (gromakovsky) [inline docs/usage-gui.md]: Could you add it in this PR please?

File: docs/usage-gui.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+make sure that you wrote that mnemonic somewhere safe. To verify that, you will be asked to first
+write down the mnemonic, e.g. to a piece of paper, and then type it back into Ariadne. If the
+mnemonic matches, the wallet will be actually created. If, however, you fail to confirm your
+mnemonic, you have to start the process from the beginning.
+
+Once the wallet is created you can select it in the wallet tree. On the right you will see its name
+and balance. Also there will appear a "Create account" button. To start actually using the wallet
+you need to create at least one account.
+
+**TODO** AD-296 Create an account and an address in new wallet/account automatically
```
Comment (gromakovsky) [inline docs/usage-gui.md]: AD-296 is done.

**Comment by gromakovsky** [comment]
I think PR template should be updated, btw.

**Comment by maksbotan** [comment]
Updated everything.

**Comment by gromakovsky** [comment]
Except README :)

**Pull request merged**